src/game/server/gamemodes/insta_core/insta_core.h

```diff
@@ -175,5 +176,9 @@ class CGameControllerInstaCore : public CGameControllerDDRace
 	// UniqueId unlike regular ClientIds are not reused
 	// they start at one and get incremented for every new player that gets created
 	CPlayer *GetPlayerByUniqueId(uint32_t UniqueId);
+
+	const char *GetMysteryRoundLine();
+	static std::mt19937 M_S_RANDOM_ENGINE;
+	static size_t M_S_LAST_LINE;
```

Comment:
Can we make this a member instead of a static variable and give it a nicer name like `m_LastMysteryLine`